### ISSUE TITLE
Default output is too obscure

### ISSUE THREAD

**Issue by tommiv** [hapijs/lab#970]
#### Support plan

* *which support plan is this issue covered by?*
community
* *is this issue currently blocking your project?*
no
* *is this issue affecting a production system?*
no

#### Context

* *node version*: v12.9.1
* *module version*: ^22.0.3
* *environment* (e.g. node, browser, native): node
* *used with* (e.g. hapi application, another framework, standalone, ...): standalone via npm script
* *any other relevant information*: –

#### How can we help?

Let's say I have some test scripts in `test/*.js`. When I run

```
npm run test
```
or
```
npx lab
```

all I get in terminal is a dot (literally, as many dots as many tests I have) and the report, something like

```
  .......

7 tests complete
Test duration: 112 ms
Leaks: No issues
```

Is this intended behavior? How could I get more detailed report with experiments/tests names? Now they are shown only for failed tests. I don't see any cli flag for this in documentation.

**Comment by cjihrig** [comment]
You just have to pass the `-v` argument to get better output.

**Comment by tommiv** [comment]
Oh. I got it. I've tried `--verbose` variant before I created this ticket. I usually prefer long options to improve readability. The thing is, when you call it like this

```
npm test --verbose
```

the `--verbose` flag comes to npm itself. Output changes, but it's not what I want, obviously.

The solutions is explicilty bypass this flag to the `lab` binary, like this:

```
npm run test -- -v
```

or 

```
npm run test -- --verbose
```
